maven/poms.bzl

```diff
@@ -43,17 +43,18 @@ def _parse_fragment(deps_fragment):
         elif key == "systemPath":
             systemPath = token
 
-    return struct(
+    dependency_struct = struct(
         group_id = group_id,
         artifact_id = artifact_id,
         version = version,
-        type = "jar",
-        optional = False,
-        scope = "compile",
-        classifier = None,
-        system_path = None,
+        type = type,
```

Comment:
_snort_

Reply:
I know what happened too.  I got totally mixed up between assignements to a struct() and declarations of function parameters.  The latter case this owuld totally be fine, since it's defaults.  But a struct is actual assignment. 

Testing for the win... um... except... um... oops.

Reply:
#iblamepython